content/docs/cml/examples.md

```diff
@@ -0,0 +1,8 @@
+# Examples
+
+Here are some example projects using CML.
+
+- [Basic CML project](https://github.com/iterative/cml_base_case)
+- [CML with DVC to pull data](https://github.com/iterative/cml_dvc_case)
+- [CML with Tensorboard](https://github.com/iterative/cml_tensorboard_case)
+- [CML with EC2 GPU](https://github.com/iterative/cml_cloud_case)
```

Comment:
Something is wrong with this one because it looks like this:

![image](https://user-images.githubusercontent.com/1477535/102681566-04fd5100-4188-11eb-994a-12d83462b6a3.png)

Maybe just missing an ending new line?

Reply:
Also this could be part of the index, probably.

Reply:
yeah, this a minor engine bug :) a funny one ... I guess the only thing that is needed is to add more text :)

Reply:
I can fill it out with some more text- our GitLab example repos. LIttle hacky but it'll do

Reply:
or move it to the CML index page as "case/success studies"?